app/models/miq_request.rb

```diff
@@ -420,11 +421,27 @@ def task_check_on_execute
   def execute
     task_check_on_execute
 
-    deliver_on = nil
     if get_option(:schedule_type) == "schedule"
-      deliver_on = get_option(:schedule_time).utc rescue nil
+      start_time = get_option(:schedule_time).utc rescue nil
```

Comment:
Copying @gmcculloug 's comment (https://github.com/ManageIQ/manageiq/pull/17520#discussion_r193437017)
> I would recommend that we refactor this line. The classic UI sets get_option(:schedule_time) to a ActiveSupport::TimeWithZone object which supports the utc call.
> 
> I suspect that if this is set from the API or some other means it would set the time as a string in which case the code will silently error and use nil. It is also a good opportunity to get rid of the inline rescue as well.

Reply:
:+1: Totally agree, but I would prefer to fix it in a different PR since it's an unrelated issue